src/libraries/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -143,6 +143,25 @@ public void SetAccessControl_FileInfo_FileSecurity_Success()
             fileInfo.SetAccessControl(fileSecurity);
         }
 
+        [Fact]
+        public void SetAccessControl_FileInfo_FileSecurity_Success_NameLongerThanMaxShortPath()
+        {
+            using var directory = new TempAclDirectory();
+
+            const int MaxShortPath = 260;
+            int fileNameLength = Math.Max(MaxShortPath - directory.Path.Length, 1);
+
+            string path = Path.Combine(directory.Path, new string('1', fileNameLength) + ".txt");
```

Comment:
if I read right this is intended to be just ".txt" longer than 260? why not just make it 500 or something huge?

Reply:
FWIW: The path reported in the issue is 261, that's probably why just above 260 was chosen. 
```
C:\ProgramData\UET\SDKs\Android-android-33-33.0.1-3.10.2.4988404-25.1.8937393-jdk-11.0.19+7\Sdk\cmdline-tools\lib\external\com\google\guava\listenablefuture\9999.0-empty-to-avoid-conflict-with-guava\listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar
```

It doesn't hurt to test both 500 and > 260.